src/App.jsx

```diff
@@ -0,0 +1,55 @@
+// import logo from './logo.svg';
+import React from 'react';
+import './App.css';
+import styled, { createGlobalStyle } from 'styled-components';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Body from './components/Body';
+
+const stylingVariables = {
+  bgColor: '#fffcde',
+  black: '#121212',
+  gray: '#545353',
+  white: '#f7f7f7',
+};
+
+export const styleContext = React.createContext();
+
+const GlobalStyle = createGlobalStyle`
+  :root {
+    font-size:16px;
+  }
+
+  @media screen and (max-width: 480px) {
```

Comment:
fascinating. 

I tell you the breakpoints at which some changes on UI/UX design may change dramatically.
Recall them.
320, 360, 420, 768 (is the min-width for the desktop), 992, 1200, 1600, 2100.

All of them are for width and in terms of pixels.

Reply:
Done. I changed the media rule to max-width:420px